src/components/RichTextEditor/utils/setMinWidthIfEmpty.ts

```diff
@@ -0,0 +1,19 @@
+import React from "react";
+import { BaseEditor } from "slate";
+import { ReactEditor } from "slate-react";
+
+import { isEditorEmpty } from "@components/RichTextEditor/utils/isEditorEmpty";
+import { getCanvasFontSize, getTextWidth } from "@components/RichTextEditor/utils/getTextWidth";
+
+export const setMinWidthIfEmpty = (
+    editor: BaseEditor & ReactEditor,
+    placeholder: string,
+    setWrapperStyle: React.Dispatch<React.SetStateAction<React.CSSProperties | undefined>>,
```

Comment:
Don't really like idea of passing this one, as the function complexity is quite low, could you move it back to RTE.tsx?

Reply:
Returning `CSSProperties | undefined` now and setting the state variable inside the component